.ci/static-checks.sh

```diff
@@ -322,6 +337,11 @@ static_check_versions()
 {
 	local db="versions.yaml"
 
+	if [ -z "$have_yamllint_cmd" ]; then
+		info "Installing yamllint"
+		install_yamllint
+	fi
```

Comment:
Do you need to re-do the `have_yamllint_cmd=$(command -v "$yamllint_cmd" || true)` again, now it is hopefully installed - otherwise, will you still fail the check below?

Reply:
@grahamwhaley , I added that step at the install_yamllint function